src/components/buttons/ButonAddCrypto.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {AddButton, AddView, ButtonText} from './style';
+
+const ButonAddCrypto = ({navigation}: any) => (
```

Comment:
typo: is ButtonAddCrypto
replace all 
![image](https://user-images.githubusercontent.com/20242580/184190903-df7ccc56-ce07-470d-b333-7e480597dcf6.png)